### ISSUE TITLE
Error when installing from Pypi

### ISSUE THREAD

**Issue by fabiogibson** [jonashaag/bjoern#106]
When installing from pypi, got the following error:

> gcc: error: unrecognized command line option ‘-fcommon-fPIC’; did you mean ‘-fcommon’?

It seems to be a missing comma between  '-fcommon' and '-fPIC' at setup.py:

```
extra_compile_args = ['-std=c99', '-fno-strict-aliasing', '-fcommon'  [Missing comma here?]
                          '-fPIC', '-Wall', '-Wextra', '-Wno-unused-parameter',
                          '-Wno-missing-field-initializers', '-g']
```

**Comment by jonashaag** [comment]
That's correct,

**Comment by jonashaag** [comment]
Would you mind sending a PR?

**Comment by jonashaag** [comment]
Fixed in fd4e97fa144ab62458056d94d116d26385a41650
